src/main/java/com/google/songgame/servlets/RoundServlet.java

```diff
@@ -18,21 +31,135 @@
   private static final String CLIENT_KEY = "d15fbbe1c77552dc5097";
   private static final String CLIENT_SECRET = "91fd789bf568ec43d2ee";
   private static final String PUSHER_APPLICATION_NAME = "song-guessing-game";
-  private static final String PUSHER_CHAT_CHANNEL_NAME = "start-game";
+  private static final String PUSHER_ROUND_CHANNEL_NAME = "start-round";
+  private static final String PUSHER_GAME_CHANNEL_NAME = "start-game";
+  private static final int TIME_OFFSET = 3000;
+  private static final int ROUND_LENGTH = 30000;
   private Pusher pusher;
+  private Gson gson;
+  DatastoreService datastore;
 
   @Override
   public void init() {
     pusher = new Pusher(APP_ID, CLIENT_KEY, CLIENT_SECRET);
     pusher.setCluster("us2");
     pusher.setEncrypted(true);
+
+    gson = new Gson();
+    datastore = DatastoreServiceFactory.getDatastoreService();
   }
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    loadRound();
     pusher.trigger(
         PUSHER_APPLICATION_NAME,
-        PUSHER_CHAT_CHANNEL_NAME,
+        PUSHER_GAME_CHANNEL_NAME,
         Collections.singletonMap("message", "Start Game"));
   }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Entity game = getCurrentGame();
+    EmbeddedEntity currentRound = (EmbeddedEntity) game.getProperty("currentRound");
+    if (!isNewGame(game)) {
+      Map<String, Object> roundMap = createRoundMap(game, currentRound);
+
+      String json = new Gson().toJson(roundMap);
+      response.getWriter().println(json);
+    }
+  }
+
+  private void loadRound() {
```

Comment:
Could you move this method below doPut(), that way all the public methods are grouped together?